src/Twig/Handler/RecordHandler.php

```diff
@@ -178,21 +178,8 @@ public function listTemplates($filter = null, $safe = false)
             return null;
         }
 
-        // Get the active themeconfig
-        $themeConfig = $this->app['config']->get('theme/templateselect/templates', false);
         $files = [];
 
-        // Check: Are the templates for template chooser defined?
-        if ($themeConfig) {
-            foreach ($themeConfig as $templateFile) {
-                if (!empty($templateFile['name']) && !empty($templateFile['filename'])) {
-                    $files[$templateFile['filename']] = $templateFile['name'];
-                }
-            }
-
-            return $files;
```

Comment:
Seriously?! If it returns here, it's no wonder the other options are missing. :fire: